x-pack/plugins/security_solution/public/common/components/event_details/table/investigate_in_timeline_button.test.tsx

```diff
@@ -113,22 +96,4 @@ describe('AddToTimelineCellRenderer', () => {
       expect(screen.queryByLabelText(ACTION_INVESTIGATE_IN_TIMELINE)).not.toBeInTheDocument();
     });
   });
-
-  describe('When the field is the host status field', () => {
```

Comment:
This test is not required anymore since no prevalence data is returned for this field and the button therefore is not rendered.